datastore/datastore/go.mod

```diff
@@ -6,6 +6,7 @@ require google.golang.org/grpc v1.64.0
 
 require (
 	github.com/cridenour/go-postgis v1.0.0
+	github.com/golang/protobuf v1.5.4
```

Comment:
Ok, so removing that line from go.mod then? Or?

Reply:
I don't know enough about how go tooling works. But for me it gets removed, while for you it gets added.
Any idea? If not, we can leave it is in your branch.

Reply:
No idea either. I guess it's related to you and me having slightly different Go versions. Mine is:
```
$ go version
go version go1.22.5 linux/amd64
```
But there might be more it than that. I suggest we live the ping-ponging until we prioritize fixing it properly.

Reply:
The problem is that I have a too old protoc (3.12.4) which generates this line in `datastore.pb.go`:
```
timestamp "github.com/golang/protobuf/ptypes/timestamp"
```
instead of this:
```
timestamppb "google.golang.org/protobuf/types/known/timestamppb"
```
which in turn causes `go mod tidy` to add `github.com/golang/protobuf` to `go.mod` in my dev environment. And I have to use go mod tidy, otherwise vscode gets unhappy.

So I'll try to upgrade protoc to something newer (currently it's v3.21.12 that gets installed in the docker image built from `datastore/datastore/Dockerfile`).

Reply:
Fixed (I hope!)

Reply:
Good find!